packages/protocol/engine/tangle/blockdag/blockdag.go

```diff
@@ -139,6 +143,16 @@ func (b *BlockDAG) markSolid(block *Block) (err error) {
 	return nil
 }
 
+func (b *BlockDAG) checkTimestampMonotonicity(block *Block) error {
+	for _, parentID := range block.Parents() {
+		parent, parentExists := b.Block(parentID)
+		if parentExists && parent.IssuingTime().After(block.IssuingTime()) {
```

Comment:
I think we need to be more precise here. What's the exact rule? Issuing time of a child needs to be greater than parents? Then it should be 
`!parent.IssuingTime().Before(block.IssuingTime())` or
`!block.IssuingTime().After(parent.IssuingTime())`

Reply:
fixed.